### PR TITLE
Feat/invert option for filtering

### PR DESCRIPTION
**Pull request by csnktms**

### CODE REVIEW

**Comment by warrenday** [comment]
Hey @csnktms thanks for raising the PR. This feature looks really useful.

Could we leave revert enabled at all times, as this aligns with the chrome devtools functionality? You may also want to start with a reverted search before typing.

Could we add some tests to check:
1. Clicking invert does invert the filter

**Comment by csnktms** [comment]
hey @warrenday sorry, I was busy as well, sure let me finish it up during the weekend.

File: src/containers/Main/Main.test.tsx [hunk 1]
```diff
@@ -210,4 +210,33 @@ describe("Main", () => {
       }
     });
   });
+  it.only("filters network data with the given search query inverted", async () => {
```
Comment (warrenday) [inline src/containers/Main/Main.test.tsx:213]: Please remove .only

Comment (csnktms) [reply src/containers/Main/Main.test.tsx:213]: nice catch thanks, I was just testing it and didn't notice that left it like this.

Comment (csnktms) [reply src/containers/Main/Main.test.tsx:213]:
```suggestion
  it("filters network data with the given search query inverted", async () => {
```

**Review by warrenday — APPROVED** [review]
Great work @csnktms

**Pull request merged**